src/PacchettiBotti/Registry/Bower.hs

```diff
@@ -26,6 +38,20 @@ import qualified PacchettiBotti.Static as Static
 
 type Expr = Dhall.DhallExpr Dhall.Import
 
+type NativeDependencies = Map Text Text
+
+data NpmPackageJson =
+  NpmPackageJson
+    { npmDependencies :: NativeDependencies
+    , npmDevDependencies :: NativeDependencies
+    }
+
+instance FromJSON NpmPackageJson where
+  parseJSON = Json.withObject "npm package.json" $ \package ->
+    NpmPackageJson
+      <$> package .:! "dependencies" .!= mempty
+      <*> package .:! "devDependencies" .!= mempty
```

Comment:
Do we have to parse the `package.json` at all? Can we just parse the files and get packages from there?

Reply:
Can we infer the dependencies versions without parsing the package.json?

Reply:
Oh great point